### DISCUSSION TITLE
Air Android : flash.media.Video causes temporary black screen

### DISCUSSION THREAD

**Discussion by pol2095** [airsdk/Adobe-Runtime-Support#1651] [Q&A] [answered]
Hello,

on Android `flash.media.Video` causes temporary black screen (one second) when I add this line on a Pixel 3  :

```
var video:Video = new Video(250, 250);
```

Thanks

**Comment by MalacTheLittle** [comment, marked as answer]
Have you added `containsVideo` in app descriptor file in android section?
`<containsVideo>true</containsVideo>`

**Comment by pol2095** [comment]
no

**Reply by MalacTheLittle** [reply]
Try adding that line, it should solve the problem.

**Comment by pol2095** [comment]
Thanks, it work.
